auth_supplier/controllers/main.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp.addons.auth_signup.controllers.main import AuthSignupHome
+from openerp.http import request
+
+
+class AuthSignupHome(AuthSignupHome):
+
+    def do_signup(self, qcontext):
```

Comment:
We need overwrite this function to get account_type from qcontext otherwise
_signup_with_values only recibe 'login', 'name' and 'password'.